packages/documentation/src/stories/components/textarea/textarea.snapshot.stories.ts

```diff
@@ -0,0 +1,55 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta from './textarea.stories';
+import { html } from 'lit';
+import { bombArgs } from '../../../utils/bombArgs';
+
+export default {
+  ...meta,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const Textarea: Story = {
+  render: (_args: Args, context: StoryContext) => {
+    const longText =
+      'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.';
+    const shortText = 'Loremipsum';
+    let counter = 0;
+    return html`
+      <div>
+        ${['white', 'dark'].map(
+          bg => html`
+            <div class=${'row bg-' + bg}>
+              ${bombArgs({
+                label: [shortText, longText],
+                floatingLabel: [true, false],
+                hiddenLabel: [true, false],
+                size: context.argTypes.size.options,
+                disabled: [true, false],
+                hint: [shortText, longText],
+                validation: context.argTypes.validation.options,
+              })
+                .filter(
+                  (args: Args) => !(args.hiddenLabel === true && args.floatingLabel === false),
+                )
+                .filter(
+                  (args: Args) => !(args.hint === longText && args.validation === 'is-invalid'),
+                )
+                .filter(
+                  (args: Args) => !(args.label === longText && args.validation === 'is-invalid'),
+                )
+                .map((args: Args) => {
```

Comment:
Reduce duplicates even further, there are more than 10 examples of the valid textarea.

<img width="745" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/8f3b2c1d-35df-4d70-8afe-3642cc9196bc">